sotodlib/utils/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2018-2020 Simons Observatory.
+# Full license can be found in the top level "LICENSE" file.
+"""Simons Observatory utilities.
+
```

Comment:
Thanks for adding this missing file- slipped through the cracks in the reorganization